### ISSUE TITLE
update README.md to provide useful instructions

### ISSUE THREAD

**Issue by pedrogarcia** [JuliaIO/MAT.jl#120]
I strongly feel like Julia would be less annoying and more palatable for people considering it as a possible replacement for matlab if REPL code examples worked without requiring much knowledge about the workings of the language itself.

On a Julia installation, the very first example on the README.md reads:

```
Within Julia, use the package manager:

Pkg.add("MAT")
See also the requirements for the HDF5 module, used for "v7.3" files and for writing *.mat files.

```
When running that command in the REPL, I get:

```
PS C:\Users\phg> julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.2.0 (2019-08-20)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> Pkg.add("MAT")
ERROR: UndefVarError: Pkg not defined
Stacktrace:
 [1] top-level scope at REPL[1]:1
```

Could the installation instructions be changed to ``` using Pkg; Pkg.add("MAT")``` instead?

**Comment by timholy** [comment]
Agreed 100%. But believe it or not, it's surprisingly hard for experts to write docs that are good for newbies. The surest way to make sure such changes happen is for newbies to lend their invaluable perspective by helping out:
- in the "home page," click on the `README.md` file
- In the upper right corner, click the pencil icon
- In your web browser, make the changes you'd like to see
- At the bottom in "Propose file change," write a brief summary of your change and then submit
- Hopefully :smile:, someone with merge privileges will review your suggested change. S/he may make suggestions for further improvements or just merge outright.

**Comment by timholy** [comment]
I will make this change for you since you took the time to file it as an issue, but please apply your perspective to other places you see problems!
